enclave-runner/src/usercalls/mod.rs

```diff
@@ -605,94 +1067,42 @@ pub trait UsercallExtension : 'static + Send + Sync + std::fmt::Debug {
     ///
     /// The enclave must not make any security decisions based on the local address received.
     #[allow(unused)]
-    fn bind_stream(&self,
-                  addr: &str,
-                  local_addr: Option<&mut String>
-                  ) -> IoResult<Option<Box<dyn SyncListener>>> {
+    fn bind_stream(
+        &self,
+        addr: &str,
+        local_addr: Option<&mut String>,
+    ) -> IoResult<Option<Box<dyn AsyncListener>>> {
         Ok(None)
     }
-
 }
 
 impl<T: UsercallExtension> From<T> for Box<dyn UsercallExtension> {
-    fn from(value : T) -> Box<dyn UsercallExtension> {
+    fn from(value: T) -> Box<dyn UsercallExtension> {
         Box::new(value)
     }
 }
 
 #[derive(Debug)]
 struct UsercallExtensionDefault;
-impl UsercallExtension for UsercallExtensionDefault{}
+impl UsercallExtension for UsercallExtensionDefault {}
 
 #[allow(unused_variables)]
```

Comment:
Please remove this annotation and fix all unused variable warnings in the impl block.